config/settings/local.py

```
@@ -18,6 +18,7 @@
 
 CSRF_TRUSTED_ORIGINS = [
     "https://8000-idx-tani-pintar-backend-1735536156532.cluster-nx3nmmkbnfe54q3dd4pfbgilpc.cloudworkstations.dev",
+    "https://8000-firebase-tani-pintar-backendgit-1748227150875.cluster-ubrd2huk7jh6otbgyei4h62ope.cloudworkstations.dev"
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)